app/js/components/SearchBar.js

```diff
@@ -40,6 +40,7 @@ export default class SearchBar extends Component {
           ref={input => { this.input = input; }}
           onChange={onInputChange}
           value={searchInput}
+          autofocus=autofocus
```

Comment:
Ah React...

Okay. Since we talked about this offline, I'll just close this PR for now. 

In the case it gets reopened/reconsidered, I think there could be some sort of check on the deviceType to make sure the UX is solid on mobile.

DeviceType checking is probably some utility, but perhaps it could be as simple as...

```
componentDidMount() {
  if (isMobile()) {
    this.input.focus();
  }
}
```